enos/enos-scenario-autopilot.hcl

```diff
@@ -78,11 +90,8 @@ scenario "autopilot" {
   }
 
   step "create_vault_cluster" {
-    module = module.vault_cluster
-    depends_on = [
-      step.create_vpc,
-      step.build_vault,
-    ]
+    module     = module.vault_cluster
+    depends_on = [step.create_vpc]
```

Comment:
This can be racy. If we're using a local build variant or artifactory it's possible we try to install vault on the machines before the artifact has been built or downloaded. We should still depend on `build_vault`. Pretty sure we actually saw that happen and thus the dependency was added.

Reply:
Added back